ui-participant/src/hub/HubPage.tsx

```diff
@@ -69,7 +100,36 @@ export default function HubPage() {
           className="hub-dashboard py-4 px-2 px-md-5 my-md-4 mx-auto shadow-sm"
           style={{ background: '#fff', maxWidth: 768 }}
         >
-          {enrollees.map(enrollee => <StudySection key={enrollee.id} enrollee={enrollee} portal={portal} />)}
+          <div className="dropdown hub-dashboard-background flex-grow-1">
+            <button
+              className="btn btn-outline-primary dropdown-toggle w-100 position-relative d-flex justify-content-center
+              align-items-center" type="button"
+              data-bs-toggle="dropdown" aria-expanded="false" id="dropdownMenuButton">
+              Participants and Tasks
```

Comment:
Should we hide the account selector if an enrollee with no proxy relations is logged in?